baanders2021reproducibility/openreview/main.tex

```diff
@@ -0,0 +1,54 @@
+\documentclass{article}[12pt]
+\input{packages.tex}
+
+\newif{\ifhidecomments}
+\newcommand{\descr}[1]{\textcolor{NavyBlue}{\textbf{\textit{#1}}}}
+\newcommand{\TODO}{\textbf{\textcolor{red}{TODO}}}
+\newcommand{\TODOtext}[1]{\textcolor{red}{[[TODO: \textit{#1}]]}}
+\newcommand{\NR}[1]{\textbf{\textcolor{orange}{[[NR: #1]]}}}
+\newcommand{\CAG}[1]{\textbf{\textcolor{orange}{[[CAG: #1]]}}}
+\newcommand{\PB}[1]{\textbf{\textcolor{orange}{[[PB: #1]]}}}
+\newcommand{\TT}[1]{\textbf{\textcolor{orange}{[[TT: #1]]}}}
+\newcommand{\VrAanA}[1]{\textbf{\textcolor{green}{[[Vraag aan Angelo: #1]]}}}
+
+
+\newcommand{\file}[1]{\texttt{#1}}
+\newcommand{\revMNIST}{\textit{Color Reverse MNIST }}
+\newcommand{\USPSMNIST}{\textit{MNIST-USPS }}
+\newcommand{\rmn}{\textit{Color Reverse MNIST }}
+\newcommand{\umn}{\textit{MNIST-USPS }}
+\newcommand{\mn}{\textit{MNIST }}
+\newcommand{\usps}{\textit{USPS }}
+\newcommand{\og}{\citet{Li_2020_CVPR} }
```

Comment:
Please copy the lines 4 to 22 into package.tex. These new commands are not accessible to `make` from journal file as we only import `content.tex` and `packages.tex`, but not `main.tex`.